Formula/joplin-cli.rb

```diff
@@ -0,0 +1,24 @@
+require "language/node"
+
+class JoplinCli < Formula
+  desc "Note taking and to-do application with synchronization capabilities"
+  homepage "https://joplinapp.org/"
+  url "https://registry.npmjs.org/joplin/-/joplin-2.2.2.tgz"
+  sha256 "15b932b51632dbcf8dfa0c39541af49b7e072b1a7c204e304d0fbdea6f1d787a"
+  license "MIT"
+
+  depends_on "node"
```

Comment:
The old formula had these dependencies:

https://github.com/Homebrew/homebrew-core/blob/8771daab6cd2069533f4ec5e024d0846bb857197/Formula/joplin.rb#L20-L23

It seems `vips` at least, is still needed. If the others are still needed, we should add those dependencies too. We don't want this installing its own `vips`. See our [policy](https://docs.brew.sh/Acceptable-Formulae#stuff-that-requires-vendored-versions-of-homebrew-formulae) on vendored dependencies.